src/main/java/seedu/duke/model/item/Expense.java

```diff
@@ -0,0 +1,73 @@
+package seedu.duke.model.item;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+// @@author GuoAi
+
+/**
+ * Represents an expense item.
+ */
+public class Expense extends Item {
+
+    private final double value;
+    private String currency;
+    private LocalDate date;
+    private LocalDate now = LocalDate.now();
```

Comment:
Consider creating the variable here and setting it in the constructor instead.

Reply:
Sorry I did not get it. Which variable do you mean?

Reply:
I was thinking maybe declare the variable `now` in this scope, but put `now = LocalDate.now()` in the constructor. But this seems fine too.

Reply:
Sure I think assigning values in constructor makes more sense!